src/ZkFold/Symbolic/Compiler/ArithmeticCircuit/MonadBlueprint.hs

```diff
@@ -90,10 +92,10 @@ class Monad m => MonadBlueprint i a m | m -> i, m -> a where
     input :: m i
 
     -- | Returns a circuit with supplied variable as output.
-    output :: i -> m (ArithmeticCircuit a)
+    output :: Vector n i -> m (ArithmeticCircuit n a)
```

Comment:
I feel like now we do not use this method anywhere and we better remove it so we can have pure instances. Maybe in one of the next PRs